src/main/java/com/keboola/tableexporter/MetaFetcher.java

```diff
@@ -199,6 +199,20 @@ private String tableListingQuery() {
                 "    )\n" +
                 "    REFCOLS ON COLS.TABLE_NAME = REFCOLS.TABLE_NAME\n" +
                 "        AND COLS.COLUMN_NAME = REFCOLS.COLUMN_NAME";
+
+        if (tables.size() > 0) {
+            sql += "\nWHERE ";
+            for (int i = 0; i < tables.size(); i++) {
+                if (i > 0) {
+                    sql += " OR ";
+                }
+                sql += "(";
+                sql += "TABS.TABLE_NAME = '" + tables.get(i).getTableName() + "' AND ";
```

Comment:
Not sure this qualifies as identifier quoting.

Reply:
We can probably ship it like this, but definitely it should be fixed.

Reply:
yeah, should use prepared statement and the whole method would have to return `Statement`